patches/tModLoader/Terraria/ID/ProjectileID.TML.cs

```diff
@@ -9,6 +9,12 @@ partial class Sets
 		/// </summary>
 		public static bool[] SingleGrappleHook = Factory.CreateBoolSet(false, 13, 315, 230, 231, 232, 233, 234, 235, 331, 753, 865, 935);
 
+		// Values taken from Player.ItemCheck_Shoot type == 3029 check
+		/// <summary>
+		/// Set of Projectile IDs that will fire 1 less from <see cref="ItemID.DaedalusStormbow"/>
```

Comment:
Documentation error. The code has a 1/3 chance to fire 2 arrows instead of 3, per shot, and regular arrows have a 1/3 chance to fire 4 arrows instead of 3. So the average arrows/shot for this set is 8/3 instead of 10/3 for a relative difference of 20% fewer arrows.

Reply:
I was aware of the mechanics, but felt like exactly describing the effect would be too verbose and prone to outdated-ness if vanilla changes anything about it. How about something more vague: `Set of Projectile IDs that, when fired from Daedalus Stormbow, will have the amount reduced`. Or would just removing the `1` be enough?

Reply:
Yeah, `less` or `20% less` would suffice.

Reply:
Documentation can always be updated as it becomes inaccurate with 1.4.5 (though at this time, there are no changes to this mechanic)